libs/providers/flagd/src/lib/service/in-process/grpc/grpc-fetch.ts

```diff
@@ -62,27 +65,33 @@ export class GrpcFetch implements DataFetch {
       this._logger?.debug('Received sync payload');
       dataFillCallback(data.flagConfiguration);
       connectCallback();
+      changedCallback([]); // flags changed list not supported
```

Comment:
We weren't running a change handler, so though we got data, we never emitted flags changed.